tools/lsp/common.rs

```diff
@@ -141,6 +168,15 @@ pub struct Diagnostic {
     pub level: String,
 }
 
+#[allow(unused)]
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+pub struct ComponentAddition {
+    pub component_type: String,
+    pub import_path: Option<String>, // Url fails to convert reliably:-/
```

Comment:
If that is the import path that will show verbatim in the `import .. from` statement, that comment is misleading.